pkgs/tools/X11/wmutils-opt/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, libxcb }:
+
+stdenv.mkDerivation rec {
+  name = "wmutils-opt-${version}";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "wmutils";
+    repo = "opt";
+    rev = "v${version}";
+    sha256 = "0mfhpja2l592i2a3j2l0lg65jhsdw1b57jnv2l0759ngpl4v82zk";
+  }
+
+  buildInputs = [ libxcb ];
+
+  installFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
+    description = "Optional addons to wmutils";
+    homepage = https://github.com/wmutils/opt;
+    license = licenses.isc;
+    maintainers = with stdenv.lib.maintainers; [ vifino ];
```

Comment:
You have already done `with stdenv.lib;`, so you could do `with maintainers;` here.